CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Palm dbt Changelog
 
+## 0.0.5
```

Comment:
This should have been `v0.1.0` - I'll update before releasing

Reply:
oops lol